cmd/erasure-object.go

```diff
@@ -608,19 +609,36 @@ func (er erasureObjects) putObject(ctx context.Context, bucket string, object st
 
 		// If we have offline disks upgrade the number of erasure codes for this object.
 		parityOrig := parityDrives
+
+		var atomicParityDrives int64
```

Comment:
Still not aligned.

Reply:
@klauspost why would that be needed it's a separate variable?